doc/modules/metrics.rst

```diff
@@ -93,6 +93,46 @@ is equivalent to :func:`linear_kernel`, only slower.)
       Information Retrieval. Cambridge University Press.
       https://nlp.stanford.edu/IR-book/html/htmledition/the-vector-space-model-for-scoring-1.html
 
+.. _gower_distances:
+
+Gower distances
+-----------------
+The function :func:`gower_distances` computes the distances between the
```

Comment:
somehow the reference here is not resolving to the functions documentation here in the build.